src/main/java/LineOfBill.java

```diff
@@ -0,0 +1,15 @@
+
+// Класс для хранения данных строки счета
+public class LineOfBill {
+    private String name;
+    private float prise;
```

Comment:
🍏 Закралась опечатка)
```suggestion
    private float price;
```

Reply:
Исправил